composer.json

```diff
@@ -27,6 +27,7 @@
 		"doctrine/persistence": "^1.3.8 || ^2.2.1",
 		"nesbot/carbon": "^2.49",
 		"nikic/php-parser": "^4.13.2",
+		"ocramius/package-versions": "*",
```

Comment:
We need this to skip some test when running Doctrine 2.11

Reply:
There's now a native Composer feature for this so we don't need a package I think? https://getcomposer.org/doc/07-runtime.md

Reply:
Great, didn't know that. I've updated the PR to use this instead.

Reply:
We can remove this from composer.json now? :)

Reply:
Ah, gone too fast and forgot this. I've removed it now